lib/config/validation.ts

```diff
@@ -199,7 +201,8 @@ export async function validateConfig(
             if (key === 'fileMatch') {
               try {
                 for (const fileMatch of val) {
-                  RegExp(fileMatch);
+                  // eslint-disable-next-line no-new
+                  new RegEx(fileMatch);
                   if (!safe(fileMatch)) {
                     errors.push({
                       depName: 'Configuration Error',
```

Comment:
If I remove the package "safe-regex" then I can't have an equivalent error message saying "Unsafe regExp"